server/lib/routes/post-metrics.js

```diff
@@ -44,7 +47,7 @@ const BODY_SCHEMA = {
   entryPoint: STRING_TYPE.regex(ENTRYPOINT_PATTERN).optional(),
   entrypoint: STRING_TYPE.regex(ENTRYPOINT_PATTERN).optional(),
   events: joi.array().items(joi.object().keys({
```

Comment:
We don't, and since metrics can be sent in distinct bundles, it'd be tough to find out. Perhaps a good measure is the flow complete time?